From 3f0eb52a51057ad562fb9dedb1e61bc403f0e7ce Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.i.king@gmail.com>
Date: Mon, 6 Jan 2025 12:43:53 +0000
Subject: [PATCH] test/dsa: Fix typo "chekcing" -> "checking"
'Content-type:text/plain'

There is a typo in an info message. Fix it.

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
---
 test/dsa.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/dsa.c b/test/dsa.c
index 5638147a2c4f..59b7f477fffc 100644
--- a/test/dsa.c
+++ b/test/dsa.c
@@ -2216,7 +2216,7 @@ int batch_result_verify(struct batch_task *btsk, int bof, int cpfault)
 		}
 	} else {
 		if (core_stat == DSA_COMP_SUCCESS) {
-			info("core task success, chekcing sub-tasks\n");
+			info("core task success, checking sub-tasks\n");
 		} else if (core_stat == DSA_COMP_BATCH_PAGE_FAULT) {
 			info("batch desc list page fault\n");
 		} else if (core_stat == DSA_COMP_BATCH_FAIL) {
-- 
2.48.0

